src/components/settings/settings/EditSettingsForm.jsx

```diff
@@ -44,6 +44,10 @@ const messages = defineMessages({
     id: 'settings.app.headlineGeneral',
     defaultMessage: 'General',
   },
+  headlineRecipes: {
+    id: 'settings.app.headlineRecipes',
+    defaultMessage: 'Recipes',
```

Comment:
As far as the user is concerned, the messaging was always "Services". Should we change to that?